conceptual.md

```diff
@@ -0,0 +1,49 @@
+### Conceptual Exercise
+
+Answer the following questions below:
+
+- What is React? When and why would you use it?
+    - React is a front-end frame work.
+    - When to use: when you are building an application that implements reusable components
+    - 	Why use: it can make creating apps much simpler
+
+- What is Babel?
+	- Babel is a JavaScript transpiler that converts alternate forms of JavaScript (e.g., JSX) into regular JavaScript.
+
+- What is JSX?
+    - JavaScript XML - makes it easier to write/add/return HTML within React.
+
+- How is a Component created in React?
+	- We can creating functional components using JSX. For example, if we are creating a card component, we import react, create a function called Card that uses JSX to return the html for a component, and then export Card to be so it can be used by a different file.
+
+- What are some difference between state and props?
+	- Key difference - props are immutable while state can (and is designed to) change
+	- Props are moved down while state is typically moved up
+
+- What does "downward data flow" refer to in React?
+	- We can continue to pass down props further and further down from component to component.
+
+- What is a controlled component?
+	- A component whose state is controlled by React state.
```

Comment:
Usually refers to html elements. Like you'd only allow the value of a form input to change via react as opposed to letting the browser manage it.